incubator/mssql/values.yaml

```diff
@@ -0,0 +1,34 @@
+## mssql image version
+## ref: https://hub.docker.com/r/microsoft/mssql-server-linux/tags/
+##
+imageTag: "rc2"
+
+## Specify password for sa user
+##
+## Default: random 10 character string
+# saPassword: testing
+
+
+## Specify an imagePullPolicy (Required)
+## It's recommended to change this to 'Always' if the image tag is 'latest'
+## ref: http://kubernetes.io/docs/user-guide/images/#updating-images
+##
+imagePullPolicy: IfNotPresent
+
+## Persist data to a persitent volume
+persistence:
+  enabled: true
+  ## If defined, volume.beta.kubernetes.io/storage-class: <storageClass>
+  ## Default: volume.alpha.kubernetes.io/storage-class: default
+  ##
+  # storageClass:
+  accessMode: ReadWriteOnce
+  size: 8Gi
+
+## Configure resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+##
+resources:
```

Comment:
We've started recommending not to specify defaults for resources and leave this a conscious choice of the user. This makes it easier to test charts on systems with little resources, such as Minikube.

Do it like so:

```yaml
resources: {}
#  requests:
#    memory: 1024Mi
#    cpu: 100m
```